packages/patternfly-4/react-core/src/components/Modal/Modal.tsx

```diff
@@ -50,6 +52,7 @@ export class Modal extends React.Component<ModalProps, ModalState> {
   static defaultProps = {
     className: '',
     isOpen: false,
+    targetDocument: window.document,
```

Comment:
default to something like `document.body` then we don't have to hardcode `body` below and also give user more flexibility to append it anywhere

Reply:
Also probably have to check first that `window` (or `document` if you change it to `document.body` is not undefined, i.e.
`appendTo: typeof document !== 'undefined' && document.body`